readme.md

```diff
@@ -23,10 +23,12 @@ How you acquire typeahead.js is up to you:
 
 * Install with [npm](https://www.npmjs.com): `$ npm install corejs-typeahead`
 
+* Install with [composer](https://getcomposer.org/): `$ composer require corejavascript/typeahead.js`
+
 * [Download zipball of latest release](https://github.com/corejavascript/typeahead.js/archive/master.zip)
 
 * Download the latest dist files individually:
-  * [bloodhound.js](https://github.com/corejavascript/typeahead.js/raw/master/dist/bloodhound.js) (standalone suggestion engine)
+  * [bloodhound.js](https://github.com/corejavascript/typeahead.js/raw/master/dist/bloodhound.jcorejavascript/typeahead.jss) (standalone suggestion engine)
```

Comment:
Did you intend to change this? That URL is 404 for me. Looks like maybe a find/replace got a little greedy.

Reply:
Fixed.